lib/process-services-cloud/src/lib/form/components/widgets/radio-buttons/radio-buttons-cloud.widget.html

```diff
@@ -19,4 +19,8 @@
         </mat-radio-group>
     </div>
     <error-widget [error]="field.validationSummary" ></error-widget>
+    <error-widget
+        class="adf-radio-button-failed-rest-api-message" *ngIf="isRestApiFailed"
+        required="{{ 'FORM.FIELD.REST_API_FAILED' | translate: { hostname: getRestUrlHostName() } }}">
```

Comment:
why can't we rely on field.invalid ? 

<error-widget *ngIf="!field.isValid && isTouched() ></error-widget>

If not possible in most of the widgets we have 
*ngIf="isInvalidFieldRequired() && isTouched() 


In my opinion the isInvalid should check if is touched...

Reply:
Because it's a different kind of error that depend on failed when we fetch data. We shouldn't touch that or mark is as invalid because we don't have values that we should get from REST API in that case. 

    getValuesFromRestApi() {
        this.formCloudService.getRestWidgetData(this.field.form.id, this.field.id)
            .pipe(takeUntil(this.onDestroy$))
            .subscribe((result: FormFieldOption[]) => {
                this.resetRestApiErrorMessage();
                this.field.options = result;
                this.field.updateForm();
            }, (err) => {
                this.resetRestApiOptions();
                this.handleError(err);
            });
    }

Reply:
still we should have 1 error flag, with eventually errorMsg?